packages/devtools_app/lib/src/profiler/cpu_profile_columns.dart

```diff
@@ -70,9 +70,19 @@ class TotalTimeColumn extends ColumnData<CpuStackFrame> {
 class MethodNameColumn extends TreeColumnData<CpuStackFrame> {
   MethodNameColumn() : super('Method');
 
+  static const maxMethodNameLength = 75;
+
   @override
   dynamic getValue(CpuStackFrame dataObject) => dataObject.name;
 
+  @override
+  String getDisplayValue(CpuStackFrame dataObject) {
+    if (dataObject.name.length > maxMethodNameLength) {
```

Comment:
instead of truncating manually, we should let Flutter determine where to truncate the method name using the actual layout.